.github/workflows/scripts/get-codeowners.sh

```diff
@@ -20,13 +24,16 @@ RESULT=$(grep -c "${COMPONENT}" .github/CODEOWNERS || true)
 
 # there may be more than 1 component matching a label
 # if so, try to narrow things down by appending the component
-# type to the label
+# or a forward slash.
 if [[ ${RESULT} != 1 ]]; then
     COMPONENT_TYPE=$(echo "${COMPONENT}" | cut -f 1 -d '/')
-    COMPONENT="${COMPONENT}${COMPONENT_TYPE}"
-fi
+    OWNERS="$(get_codeowners "${COMPONENT}${COMPONENT_TYPE}")"
 
-OWNERS=$( (grep -m 1 "${COMPONENT}" .github/CODEOWNERS || true) | sed 's/   */ /g' | cut -f3- -d ' ' )
+    if [[ -z "${OWNERS:-}" ]]; then
```

Comment:
Existing code structure should be the same as before, just a single condition is added. Originally there was a single owners check because `COMPONENT` was modified to have the `COMPONENT_TYPE` appended in this case. 

So even though there's an if/else condition here now, it's the same logic.